plugins/com.google.cloud.tools.eclipse.appengine.deploy.ui/src/com/google/cloud/tools/eclipse/appengine/deploy/ui/messages.properties

```diff
@@ -9,6 +9,8 @@ deploy.login.dialog.message=Browser opened to authorize deployment.
 deploy.manual.link=To manually promote a version use the <a href="{0}">Google Cloud Console.</a>
 deploy.preferences.dialog.title=Deployment configuration
 deploy.preferences.dialog.title.withProject=Deployment configuration for {0}
+deploy.preferences.dialog.label.selectAccount=Account:
+deploy.preferences.dialog.accountSelector.login=Add a new account...
```

Comment:
This is an item in the combo box, and I recall I have seen many occasions where ankle brackets are used to distinguish this kind of instructive item from actual values in the combo list. I think this is not unusual as @briandealwis also suggested it.